### ISSUE TITLE
Bring your own data workshop for Citizen scientists in Austria, Germany, Switzerland

### ISSUE THREAD

**Issue by dagendresen** [uio-mana/Events#19]
# Bring your own data workshop for Citizen scientists in Austria, Germany, Switzerland

## Abstract
The University of Natural Resources in Vienna (BOKU) contacted the GBIF Secretariat in June to ask for help with organizing an open data workshop in Vienna in September. The topic is citizen science data and ecology. The contact points at BOKU are senior scientists Daniel Doerler and Florian Heigl.

_Notice that Austria is not a member of GBIF. Austria was an Associate non-paying member from the start in 2001 to 2013, but did not make the mandatory transition to voting paying member with the new GBIF MoU in 2012._

The workshop is planned to be organized online with participants from Austria, Germany, and Switzerland. Ideally with training in German, but English is ok. They plan a hands-on (bring your own data) style for the workshop. They have some but limited budget for the workshop. (Because Austria has chosen to not be a GBIF member I think we might want real costs to be covered - including staff time - if we would be offering to help).

**Comment by ErikKusch** [comment]
Agreed! Let's see how my new MaNa organisation plans shape up to remedy this
